From 57684272fa7f2962f39d0f3f4a03d1b0813b354f Mon Sep 17 00:00:00 2001
From: "C.r. Guo" <nxa13725@lsv07005.swis.us-cdc01.nxp.com>
Date: Fri, 27 Sep 2019 04:29:37 +0200
Subject: [PATCH] update version to 1909

---
 etc/edgescale-version | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/edgescale-version b/etc/edgescale-version
index ee099e0..32b28f9 100644
--- a/src/github.com/NXP/qoriq-edgescale-eds/etc/edgescale-version
+++ b/src/github.com/NXP/qoriq-edgescale-eds/etc/edgescale-version
@@ -1 +1 @@
-edgescale-1906
+edgescale-1909
-- 
2.7.4

